packages/loader/container-loader/src/naming.md

```diff
@@ -10,19 +10,21 @@
       * Inbound/Outbound are operational (...is there a possible race condition if `IContainerLoadMode.opsBeforeReturn === undefined`?)
   * `connected: boolean`
     * `connectionState === ConnectionState.Connected`
-  * `resume()`
-    * Calls `DeltaManager.connect`
-  * `setAutoReconnect(reconnect: boolean)`
-    * If false, will disconnect from delta stream.  If true, will probably connect to delta stream
+  * `connect()`
+    * Calls `DeltaManager.connect`  (and picks up some semantics from old `setAutoReconnect`?)
+  * `disconnect()`
+    * Disconnect from delta stream, until `connect()` is called again.
```

Comment:
Could go more explicit and do `connectToDeltaStream` and `disconnectFromDeltaStream`... might be better to spell it out.

@vladsud - I recall you talking about some different abstractions that Word is using, like it doesn't have a websocket connection or something?  Can you remind me of what that is so we can check the proposed naming/concepts against that paradigm?

Reply:
Discussed in Scott's PR and decided against it.